crates/circuits/primitives/src/is_less_than/mod.rs

```diff
@@ -159,6 +160,11 @@ impl IsLtSubAir {
             bits_remaining = bits_remaining.saturating_sub(self.bus.range_max_bits);
         }
     }
+
+    pub fn columns<F: Field, const AUX_LEN: usize>(&self) -> Vec<String> {
+        LessThanAuxCols::<F, AUX_LEN>::flatten_fields().unwrap()
+        // IsLessThanIO ?
```

Comment:
This case seems to define only one struct Cols for auxiliaries (`LessThanAuxCols`), so I have my doubts if the real information is actually the one from `IsLessThanIO`. In that case, should I return both? Only `IsLessThanIO`?